pkg/apis/core/v1/helper/helpers_test.go

```diff
@@ -1398,3 +1398,33 @@ func TestNodeSelectorRequirementKeyExistsInNodeSelectorTerms(t *testing.T) {
 		}
 	}
 }
+
+func TestHugePageUnitSizeFromByteSize(t *testing.T) {
+	tests := []struct {
+		size     int64
+		expected string
+	}{
+		{
+			size:     1024,
+			expected: "1KB",
+		},
+		{
+			size:     33554432,
+			expected: "32MB",
+		},
+		{
+			size:     3221225472,
+			expected: "3GB",
+		},
```

Comment:
add a unit test covering the error case and the "valid > 1024 of the largest unit" case

Reply:
thx for review. updated it.